ui/v2.5/src/models/list-filter/types.ts

```diff
@@ -22,6 +22,12 @@ export interface ILabeledValue {
   value: string;
 }
 
+export function encodeILabeledId(o: ILabeledId) {
+  const ret = { ...o };
+  ret.label = encodeURIComponent(o.label);
+  return ret;
+}
```

Comment:
You can just do `return { ...o, label: encodeURIComponent(o.label) }`. Properties are set in the order they are passed in, so even if `o` has a label, it will be overwrittenby encodeURIComponent.

Reply:
Done.